components/navigation/navigation.css

```diff
@@ -0,0 +1,61 @@
+.nav-header {
+    background-color: var(--color-primary);
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+  
+  .nav-hamburger {
+    padding: var(--space-sm);
+    color: var(--color-white);
```

Comment:
Its good to see that you have consistently used var throughout.

Reply:
Thanks for the review. :)